### PR TITLE
s@SDDDdfsdasss

### PR DESCRIPTION
**Pull request by MadhurangaWije**
ssad

### CODE REVIEW

**Comment by ravindra-wije** [comment]
sdfsd

**Review by ravindra-wije — COMMENTED** [review]
this is review comment feedbackasffaas

**Review by MadhurangaWije — COMMENTED** [review]
adsadfsadf

**Review by MadhurangaWije — COMMENTED** [review]
asas

**Review by MadhurangaWije — COMMENTED** [review]
asdasd

**Review by MadhurangaWije — COMMENTED** [review]
asd

**Review by MadhurangaWije — COMMENTED** [review]
adasf

**Review by MadhurangaWije — COMMENTED** [review]
sdsdf

**Review by MadhurangaWije — COMMENTED** [review]
adfasf

1. sdf
2. sdf
3. sdf
4. sdfs
5. df
6. s
7. dfs
8. df

**Review by ravindra-wije — COMMENTED** [review]
pppppppppppppppppppppp

**Review by ravindra-wije — CHANGES_REQUESTED** [review]
change this file

**Review by ravindra-wije — COMMENTED** [review]
sdfsd

**Comment by MadhurangaWije** [comment]
> sdfsd

what do i mean by this

**Comment by MadhurangaWije** [comment]
> sdfsd

jis
s
ss
s


s
s
s

s

**Comment by MadhurangaWije** [comment]
> pppppppppppppppppppppp

sdfkjsd 
sdf
sdf
sd
f
sdf
sdf**sdfsdfsdfsd**

File: src/db/main.bal [hunk 1]
```diff
@@ -5,7 +5,7 @@ public function main() returns error?{
     io:println("Created MySQL client");
```
Comment (MadhurangaWije) [inline src/db/main.bal:6]: asdasd

File: src/db/main.bal [hunk 2]
```diff
@@ -5,7 +5,7 @@ public function main() returns error?{
...
 
     check dbClient.close();
-    // some changes 2
+    // some changes 22
```
Comment (MadhurangaWije) [inline src/db/main.bal:8]: asdfasf

**Comment by MadhurangaWije** [comment]
> sdfsd

sdfs
sdf
s
df
s                                                     asddddddddddddddddddddddddddddddd
df
sdm

File: src/db/main.bal (hunk 2 above)
Comment (MadhurangaWije) [inline src/db/main.bal:8]: this is pull request review comment

**Review by MadhurangaWije — COMMENTED** [review]
pr review comment

File: src/db/main.bal [hunk 3]
```diff
@@ -5,7 +5,7 @@ public function main() returns error?{
     io:println("Created MySQL client");
 
     check dbClient.close();
-    // some changes 2
```
Comment (MadhurangaWije) [inline src/db/main.bal:8]: remove this line

**Review by MadhurangaWije — COMMENTED** [review]
remove commented liens

File: src/db/main.bal (hunk 3 above)
Comment (MadhurangaWije) [reply src/db/main.bal:8]: dddd

**Pull request closed without merging**